docs/build/messages/read-receipt.mdx

```diff
@@ -297,6 +297,10 @@ if (message.contentTypeId === "xmtp.org/readReceipt:1.0") {
 
 _`ReadReceipts` have an `undefined` or `nil` fallback, indicating the message is not expected to be displayed. Refer to [how handle unsupported content types](/docs/build/messages/#handle-an-unsupported-content-type-error) section._
 
+## Notifications and read receipts
+
+Read receipts do not trigger push notifications. They contain a function `shouldPush=false` that indicates the notification should not be sent.
```

Comment:
Same here. It depends on if the push notification handles this flag. Our example does but that is not guarnteed.